File: CG_unoesc_2024/Desenho2D-Avançado/src/desenho2d/Pessoa.java
package desenho2d;

public class Pessoa {
	private String nome;
	private int idade;

	public Pessoa(String nome, int idade) {
		super();
		this.nome = nome;
		this.idade = idade;
	}

	@Override
	public String toString() {
		return "Pessoa [nome=" + nome + ", idade=" + idade + "]";
	}

	public String getNome() { return nome; }
	public int getIdade() { return idade; }
}

File: Desenho2D/Desenho2D - Avançado/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static System.Windows.Forms.VisualStyles.VisualStyleElement;
using System.Xml.Linq;

namespace Desenho2D_Básico
{
    internal class Pessoa
    {
        public string Nome { get; }
        public int Idade { get; }

        public Pessoa(string nome, int idade)
        {
            Nome = nome;
            Idade = idade;
        }

        public override string ToString()
        {
            return "Pessoa [nome=" + Nome + ", idade=" + Idade + "]";
        }
    }
}
